frontend/src/metabase/public/components/EmbedModal/EmbedModal.tsx

```diff
@@ -35,14 +41,29 @@ export const EmbedModal = ({
   onClose: () => void;
   children: ({
     embedType,
-    setEmbedType,
+    goToNextStep,
+    goToPreviousStep,
   }: {
     embedType: EmbedModalStep;
-    setEmbedType: (type: EmbedModalStep) => void;
+    goToNextStep: () => void;
```

Comment:
I'd much rather have these `goTo` functions as explicitly `goToNext` and `goToPrevious` since we're going to need to add some logic to the `goToNext` function, while `goToPrevious` will always return us to the landing. I didn't want to reuse `setEmbedType` since someone else might look at this code and have no idea what `setEmbedType("application")` or `setEmbedType("legalese")` means at first glance

Reply:
Yeah I think that's a good call. I'll change it to `goBackToEmbedModal` or something like that